pytorch_lightning/loops/base.py

```diff
@@ -198,16 +198,17 @@ def state_dict(self, destination: Optional[Dict] = None, prefix: Optional[str] =
 
     def load_state_dict(
         self,
-        state_dict: Dict,
+        state_dict: Optional[Dict] = None,
```

Comment:
why does this need to be optional?

Reply:
let's see if the below enforcement is required or not. Will update this, once that gets finalized.